post-scan-actions/aws-python-slack-notification/terraform/iam.tf

```diff
@@ -0,0 +1,20 @@
+# Lambda execution role
+resource "aws_iam_role" "SlackNotificationLambdaRole" {
+  name_prefix   = "SlackNotificationLambdaRole-"
+  description = "AWS IAM Role for managing aws lambda"
+  assume_role_policy = <<EOF
```

Comment:
How about using a data source to create the policy?

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role#example-of-using-data-source-for-assume-role-policy

Reply:
Good catch! I will make that change and push an update

Reply:
That is done!